src/core/ayayaa-server/ayayaa-logging/Logger.cs

```diff
@@ -0,0 +1,45 @@
+﻿using ayayaa.logging.Enums;
+using ayayaa.logging.Interfaces;
+using ayayaa.logging.Writers;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ayayaa.logging
+{
+    public class Logger : ILogger
+    {
+        public Logger(LogType lType)
+        {
+            LoggingType = lType;
+
+            switch (LoggingType)
+            {
+                case LogType.Console:
+                    Writer = new ConsoleWriter();
+                    break;
+                case LogType.File:
+                    break;
+                case LogType.Remoteserver:
+                    break;
+                default:
+                    break;
```

Comment:
This line is dangerous, this would create an Logger without writer. All log output would then be silently ignored and/or possible NullReferenceExceptions.